src/helpers/WidgetHelper.js

```diff
@@ -707,18 +708,29 @@ export async function getWidgetConfig(
       }
     }
 
+    let additionalParams = {};
+    if (visualizationType === 'map') {
+      additionalParams = {
+        type: 'map',
+        layer_id: layer && layer.id,
+        zoom,
+        ...latLng,
+        ...(bounds
```

Comment:
It's only syntax, but I found the other day that you can do this instead of doing a ternary. I think you are gonna like it.
```
...!!bounds && { bbox: [bounds[0][1], bounds[0][0], bounds[1][1], bounds[1][0]] }
```

Reply:
Thanks! Didn't know about that.